File: RA1/clase12/PYTHON/Perro.py
from Animal import Animal

class Perro(Animal):
    def __init__(self, nombre, especie, raza, edad, peso, pedigri):
        super().__init__(nombre, especie, raza, edad, peso)
        self.pedigri = pedigri

    def bark(self):
        print(self.nombre + " esta ladrando...")
    

    def sleep(self):
        print(self.nombre + " es un perro que esta duermiendo.. zzz")

    
    def make_sound(self):
        print(self.nombre + " hace un guauu...")

File: RA1/clase12/JAVA/Perro.java
public class Perro extends Animal{
    private String pedigri;

    public Perro(String nombre, String especie, String raza, int edad, double peso, String pedigri){
        super(nombre, especie, raza, edad, peso);
        this.pedigri = pedigri;
    }

    public void bark(){
        System.out.println(this.nombre + " esta ladrando...");
    }

    //Metodo por sobrecarga
    public void sleep(){
        System.out.println(this.nombre + " es un perro que duerme... zzz ");
    }

    @Override
    public void makeSound(){
        System.out.println(this.nombre + " hace guauu...");
    }
}